src/database/DataTypes/Item.js

```diff
@@ -248,6 +248,8 @@ Item.schema = {
     unit: { type: 'Unit', optional: true },
     defaultRestrictedLocationType: { type: 'LocationType', optional: true },
     directions: { type: 'linkingObjects', objectType: 'ItemDirection', property: 'item' },
+    dosesPerVial: { type: 'double', default: 0 },
```

Comment:
do we like `dosesPerVial`?

Reply:
`dosesPerVial` for me

Reply:
no, wait. the `ItemBatch` has `doses` and I think the two should be consistent. I was choosing `perVial` because the specificity is helpful - but I'm now reconsidering and think that the general `doses` could be more useful